src/viam/app/app_client.py

```diff
@@ -375,6 +375,41 @@ class AppClient:
 
     Constructor is used by `ViamClient` to instantiate relevant service stub. Calls to `AppClient`  methods should be made through
     `ViamClient`.
+
+    Establish a Connection::
```

Comment:
(q) The `::` pattern appears slightly differently here, at the end of the `Establish a Connection` line rather than as its own line. Is this intentional?

Reply:
yes - that makes that appear as a title before the code snippet